Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -177,6 +177,22 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         return;
     }
 
+    if (createOptions.signal) {
+        m_isCancelling = true;
```

Comment:
This will set isCancelling to true whenever the site passes a signal. We want to set isCancelling to true whenever they call abort the signal, then cancel the request and continue it after the cancel has completed.

i.e. This `m_isCancelling = true;` call should go in the lambda passed to addAlgorithm, such that it gets called whenever the signal handler does, after the weakThis check on lines 183-184.